Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs

```diff
@@ -59,6 +67,34 @@ public void Toggle(Section section)
             }
         }
 
+        private void Expand()
+        {
+            view.CollapseButton.gameObject.SetActive(true);
+            view.ExpandButton.gameObject.SetActive(false);
+
+            RectTransform transform = (RectTransform) view.transform;
+
+            collapseExpandCancellationToken = collapseExpandCancellationToken.SafeRestart();
+
+            transform.DOAnchorPosX(0f, 1f)
+                     .ToUniTask(cancellationToken: collapseExpandCancellationToken.Token)
+                     .Forget();
+        }
+
+        private void Collapse()
+        {
+            view.CollapseButton.gameObject.SetActive(false);
+            view.ExpandButton.gameObject.SetActive(true);
+
+            RectTransform transform = (RectTransform) view.transform;
+
+            collapseExpandCancellationToken = collapseExpandCancellationToken.SafeRestart();
+
+            transform.DOAnchorPosX(470f, 1f)
```

Comment:
can this value (470f) be obtained from the rect or the initial position of the panel or wherever else? I'm thinkin in case the panel is resized in the future and we will wonder why its anchoring wrongly xD